### PR TITLE
Adding some clothing subtypes.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
## Description of changes
Adds various coloured subtypes of clothing.

## Why and what will this PR improve
Useful for lockers/mapping, needed for Polaris map conversion.

## Authorship
Scarf icons are from Polaris.

## Changelog
Nothing player-facing.

### CODE REVIEW

File: code/modules/clothing/head/jobs.dm [hunk 1]
```diff
@@ -94,7 +94,17 @@
 	desc = "A stylish, if purple, beret. For personnel that are more inclined towards style than safety."
 	color = COLOR_PURPLE
 
+/obj/item/clothing/head/beret/orange
+	name = "orange beret"
+	desc = "A stylish orange, beret. For personnel that are more inclined towards style than safety."
+	color = COLOR_PURPLE
```
Comment (PsyCommando) [inline code/modules/clothing/head/jobs.dm]: "orange beret", ``color = COLOR_PURPLE`` 🤔

Comment (MistakeNot4892) [reply code/modules/clothing/head/jobs.dm]: Sigh, I swear I fixed this twice now. I kept having to recommit these changes.

**Review by afterthought2 — DISMISSED** [review]

File: code/modules/clothing/head/jobs.dm [hunk 2]
```diff
@@ -94,7 +94,17 @@
...
+	color = COLOR_ORANGE
+
 /obj/item/clothing/head/beret/plaincolor
 	name = "beret"
 	desc = "A simple, solid color beret. This one has no emblems or insignia on it."
-	color = COLOR_WHITE
\ No newline at end of file
+	color = COLOR_WHITE
+
+/obj/item/clothing/head/beret/orange
```
Comment (PsyCommando) [inline code/modules/clothing/head/jobs.dm]: Speaking of the orange beret you got a duplicated here?

Comment (MistakeNot4892) [reply code/modules/clothing/head/jobs.dm]: God dammit.

Comment (PsyCommando) [reply code/modules/clothing/head/jobs.dm]: I guess one of them was meant to be a purple beret maybe?

**Review by afterthought2 — APPROVED** [review]

**Pull request merged**